microsoft-edge/webview2/webview2-api-reference.md

```diff
@@ -39,6 +39,14 @@ WebView2 is available for the following frameworks or platforms:
 ---
 
 
+<!-- ------------------------------ -->
+#### JavaScript
+
+On the web side of a WebView2 app, WebView2 exposes JavaScript APIs to the HTML documents that are running within the WebView2 control.  These JavaScript APIs are mostly to support communication with the host application.
+
+* [JavaScript](/microsoft-edge/webview2/reference/javascript/)
+
```

Comment:
```suggestion
* [Web APIs](https://developer.mozilla.org/docs/Web/API) at MDN.
   * [Event](https://developer.mozilla.org/docs/Web/API/Event)
```
d/k if this fits here.
d/k if this link fits in See Also h2 below.
d/k if should link to specific API items or to the top level.

Reply:
It may add confusion if adding these 2 links. It should be okay not having them.